e2e/support/helpers/e2e-custom-column-helpers.ts

```diff
@@ -18,16 +18,12 @@ export function enterCustomColumnDetails({
   name?: string;
   blur?: boolean;
 }) {
-  cy.get(".ace_text-input")
-    .first()
-    .as("formula")
-    .should("exist")
-    .focus()
-    .clear()
-    .type(formula);
+  const input = cy.get(".ace_text-input").first().as("formula");
+
+  input.should("exist").focus().clear().type(formula);
 
   if (blur) {
-    cy.get("@formula").blur();
+    input.blur();
```

Comment:
this approach seems a bit off for me. once there’s an alias, there’s no point of using the `input` variable. I think the old approach was better when it comes to Cypress syntax. On runtime I believe these do pretty much the same thing